stackdriver/sinatra_error_reporting.rb

```diff
@@ -0,0 +1,18 @@
+require "sinatra"
+
+get "/" do
+
```

Comment:
This is missing a response. At least respond with something

```ruby
get "/" do
  erb :template
end
```

Reply:
Actually, another example appears to use "hello world" so use that or make them consistent